UnicornRender/utility/sources/Math.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/utility/Math.hpp>
+
+#define GLM_ENABLE_EXPERIMENTAL
+#include <glm/gtx/norm.hpp>
+#include <glm/gtx/quaternion.hpp>
+#include <glm/gtc/quaternion.hpp>
+
+namespace unicorn
+{
+namespace utility
+{
+namespace math
+{
+
+glm::quat RotationBetweenVectors(glm::vec3 start, glm::vec3 dest, glm::vec3 worldX, glm::vec3 worldZ)
+{
+    start = glm::normalize(start);
+    dest = glm::normalize(dest);
+
+    float const cosTheta = glm::dot(start, dest);
+    glm::vec3 rotationAxis;
+
+    float const floatError = 0.001f;
```

Comment:
This constant should be in the unified head file so that the whole system would have the same degree of accuracy.